src/modules/assessments/components/AssessmentTitle.tsx

```diff
@@ -26,13 +26,16 @@ const AssessmentTitle = ({
   exitDate,
 }: AssessmentTitleProps) => {
   return (
-    <>
-      <Typography variant='h4'>{clientName}</Typography>
+    <Stack gap={1} component='h1' sx={{ m: 0 }}>
```

Comment:
I'd like to note that this change may still not be ideal, since it requires the screenreader user to now "step down" into the `h1` component if they want to click on the link to the enrollment. However, I think it's better to have all this info in the h1 -- but could be swayed about that, let me know if you disagree!